chromium/pages/popup/onion-rules.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title data-i18n="about_ext_name"></title>
+    <style>
+        table { border-collapse: collapse }
+        th, td { border: 1px solid gray }
+    </style>
+    <script src="../translation.js"></script>
+    <script src="onion-rules.js"></script>
+  </head>
+  <body>
+    <h1>Enabled Onion Rules</h1>
+    <table>
+        <thead>
+            <tr><th>From regex</th><th>Scope regex</th><th>Host</th><th>To</th></tr>
+        </thead>
+        <tbody id="onionTable"></tbody>
+    </table>
+  </body>
+  </html>
```

Comment:
Please add a newline at the end of this file.

Reply:
Done.